src/AWS4AuthRequest.jl

```diff
@@ -34,6 +34,17 @@ function request(::Type{AWS4AuthLayer{Next}},
     return request(Next, url, req, body; kw...)
 end
 
+# Normalize whitespace to the form required in the canonical headers.
+# Note that the expected format for multiline headers seems not to be explicitly
+# documented, but Amazon provides a test case for it, so we'll match that behavior.
+function _normalize_ws(s::AbstractString)
```

Comment:
Can we just fully spell that out:
```
_normalize_whitespace
```

Reply:
It's an internal function used in one place and I have explanatory comments at the definition and at the call site, so that seems unnecessary to me.

Reply:
The comment at the callsite wouldn't be required if it wasn't true that `ws` is a unclear acronym.